cluster/images/etcd-empty-dir-cleanup/Makefile

```diff
@@ -14,9 +14,9 @@
 
 .PHONY:	build push
 
-ETCD_VERSION = 2.2.1
+ETCD_VERSION = 3.0.14
 IMAGE = gcr.io/google_containers/etcd-empty-dir-cleanup
-TAG = 0.0.1
+TAG = 3.0.14
```

Comment:
potentially, though I expect busybox to be less of a security issue than other base images.

an open question is how exactly to update images when we only want to update the base image: do we use explicit tags, which requires us to update manifests, or do we just move the tag to the newer image?

I guess I can tag this 3.0.14.0 while we're trying to answer that question.